src/NuGet.Core/NuGet.ProjectModel/PackageSpecWriter.cs

```diff
@@ -65,7 +65,7 @@ internal static void Write(PackageSpec packageSpec, IObjectWriter writer, bool h
 #pragma warning restore CS0612 // Type or member is obsolete
 
 
-            if (packageSpec.Dependencies.Any())
+            if (packageSpec.Dependencies.Count > 0)
```

Comment:
Go ahead and revert these since we'll fix it in https://github.com/NuGet/NuGet.Client/pull/5629/files